packages/core/src/render3/interfaces/view.ts

```diff
@@ -78,7 +78,7 @@ export interface OpaqueViewState {
  * Keeping separate state for each view facilities view insertion / deletion, so we
  * don't have to edit the data array based on which views are present.
  */
-export interface LView extends Array<any> {
+export interface LView<T = unknown> extends Array<any> {
```

Comment:
I'd propose adding more descriptive generic name:
```suggestion
export interface LView<Context = unknown> extends Array<any> {
```

Reply:
I don't mind making the change, but it goes against the convention we have everywhere else where we use single-character names for generics.

Reply:
I personally feel clarity is better than convention in this case.